v2/trace/http-headers_test.go

```diff
@@ -9,6 +9,13 @@ import (
 	"github.com/SKF/go-utility/v2/trace"
 )
 
+func Test_AllHeaders(t *testing.T) {
+	const expectedLength = 3 + 5
```

Comment:
should we add constants in the test like:
```
const noOfB3Headers = 3
const noOfDatadogHeaders = 5
```

Reply:
I agree. Updated